foghorn-edge/templates/NOTES.txt

```diff
@@ -0,0 +1,3 @@
+Foghorn lightning edge installed successfully!
```

Comment:
License and Copyright information is missing.
open-ness/edgeapps repo hosts the open source edge applications. The license must be open source License.